### DISCUSSION TITLE
Polish sentences that should be supported

### DISCUSSION THREAD

**Discussion by synesthesiam** [OHF-Voice/intents#388] [Supported Sentences]
If you don't feel comfortable contributing to the YAML in this repository, you can post your **Polish** (`pl`) sentences that match [our supported intents](https://github.com/home-assistant/intents/blob/main/intents.yaml) below.

**Comment by synesthesiam** [comment]
@Uriziel01, @Wojciechgc, @Misiu - creating discussion here for Polish.

**Comment by Misiu** [comment]
I think we should support these types of sentences:
`otwórz wszystkie rolety w salonie`, does not work, but `otwórz salon wszystkie rolety` works fine.
what about `zamknij wszystkie rolety`? This should close blinds in entire house

**Reply by witold-gren** [reply]
Hey, I added support for this sentences in latest PR

**Comment by Misiu** [comment]
This two works: `czy jest jakaś otwarta roleta w biurze?`, `czy w biurze jest otwarta roleta?`
This doesn't: `czy roleta w biurze jest otwarta?`
Personally, the last one feels most natural to me, any ideas can we support that variant?

**Reply by Uriziel01** [reply]
@witold-gren Just a casual question - are any of the changes you are working on intended to tackle this kind of sentence? We did not synchronize for the longest time so I'm not sure what is going on right now.

**Reply by witold-gren** [reply]
Hey guys, I added support for this sentence (also you can fint this sentences in test case) https://github.com/home-assistant/intents/pull/2392/commits/c1a5645feda97605ca4101a9330902a4c85d4c8a

**Comment by difrost** [comment]
How is the area declination handled? This apparently works for certain ("predefined"?) areas, like `Salon` but not for many other. Here's an example:
`Włącz świtało w Łazienka` -- will turn on light in bathroom but the correct way to spell this in Polish would be:
`Włącz światło w Łazience` -- which doesn't work as there's no area `Łazience`. 

This misbehavior applies to quite many sentences. Another one is:
`Włącz lampę w Korytarzu` -- where `Korytarzu` (area `Korytarz` in PL) is literally `in corridor`. The Assist will respond to such query with a message that there's no such areas `w Korytarzu` but if one will say `Włącz lampę w Korytarz` (without proper declination of the area) all works fine.   

EDIT: Actually I've tested and `w Salonie` doesn't work ... Also from what I can tell the device names declination doesn't work as well: e.g.: `Lampa` -> `Lampę`. This basically renders the use of PL unusable unless one won't use English-like spelling, e.g.: `Włącz Lampa Salon` instead of correct `Włącz Lampę w Salonie`.

I'm happy to contribute - i'm trying to find out semantics of intents.

**Reply by Misiu** [reply]
I think that for now you can use aliases, it works fine for areas.
